rules/prefer-power-assert.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+var espurify = require('espurify');
+var deepStrictEqual = require('deep-strict-equal');
+
+var notAllowed = [
+	'notOk',
+	'true',
+	'false',
+	'is',
+	'not',
+	'regex',
+	'ifError'
+];
+
+var avaImportDeclarationAst = {
+	type: 'ImportDeclaration',
+	specifiers: [
+		{
+			type: 'ImportDefaultSpecifier',
+			local: {
+				type: 'Identifier',
+				name: 'test'
+			}
+		}
+	],
+	source: {
+		type: 'Literal',
+		value: 'ava'
+	}
+};
+
+var avaVariableDeclaratorAst = {
+	type: 'VariableDeclarator',
+	id: {
+		type: 'Identifier',
+		name: 'test'
+	},
+	init: {
+		type: 'CallExpression',
+		callee: {
+			type: 'Identifier',
+			name: 'require'
+		},
+		arguments: [
+			{
+				type: 'Literal',
+				value: 'ava'
+			}
+		]
+	}
+};
+
+function isTestFunctionCall(callee) {
+	return callee.type === 'Identifier' &&
+		callee.name === 'test';
+}
+
+function isTestFunctionCallWithSingleModifier(callee) {
+	return callee.type === 'MemberExpression' &&
+		callee.object.type === 'Identifier' &&
+		callee.object.name === 'test';
+}
+
+function isTestFunctionCallWithDoubleModifiers(callee) {
+	return callee.type === 'MemberExpression' &&
+		callee.object.type === 'MemberExpression' &&
+		callee.object.object.type === 'Identifier' &&
+		callee.object.object.name === 'test';
+}
```

Comment:
Any way to make these more generic? You could actually have more than two modifiers. `test.skip.serial.cb()` Would be better to support any amount of nested properties.